aviary/subsystems/energy/battery_builder.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import openmdao.api as om
+
+from aviary.subsystems.subsystem_builder_base import SubsystemBuilderBase
+from aviary.subsystems.energy.battery_sizing import SizeBattery
+from aviary.variable_info.variables import Aircraft, Dynamic
+
+
+class BatteryBuilder(SubsystemBuilderBase):
+    def build_pre_mission(self, aviary_inputs=None):
+        return SizeBattery(aviary_inputs=aviary_inputs)
+
+    def get_mass_names(self):
```

Comment:
should this be a property?

Reply:
Maybe; are you asking if `get_mass_names` should instead be `mass_names`? If so, we should revisit more of the `get_` methods in subsystem builders in general; this would be a breaking change for users with external subsystems. Let me know if that's what you meant or if I'm misinterpreting.

Reply:
```suggestion
    @property
    def get_mass_names(self):
```
so it can be accessed as `BatteryBuilder.get_mass_names` instead of `BatteryBuilder.get_mass_names()`, since no arguments are used

Reply:
Got it, I like that, will make issue for future

Reply:
If it's a property I think changing to `mass_names` will be cleanest. There are probably other functions in the builder that could be properties too.